hepcrawl/extractors/hindawi_parser.py

```diff
@@ -76,16 +80,24 @@ def get_authors(self, node):
                     'affiliations': self.get_affiliations(author),
                 })
 
+        if not authors:
+            logger.error('No authors found.')
```

Comment:
This log does not bring any value as it is. If relevant, I would add some info about the article (DOI or any other identifier), in case you expect the arxiv id. If we don't, maybe avoid logging at all...wdyt?